integration/tests/handshake.test.ts

```diff
@@ -162,7 +162,7 @@ test.describe('Client handshake @generic', () => {
     expect(res.headers.get('location')).toBe(
       `https://${config.pkHost}/v1/client/handshake?redirect_url=${encodeURIComponent(
         `${app.serverUrl}/`,
-      )}&suffixed_cookies=false&__clerk_hs_reason=session-token-expired&__clerk_refresh=no-cookie${devBrowserQuery}`,
+      )}&suffixed_cookies=false&__clerk_hs_reason=session-token-expired-refresh-non-eligible-no-cookie${devBrowserQuery}`,
```

Comment:
🔧 Shouldn't this be `session-token-expired-refresh-non-eligible-no-refresh-cookie` instead?

Reply:
Fixed! thank you!